@xen-orchestra/web/src/components/vm/dashboard/VmDashboardBackupRuns.vue

```diff
@@ -0,0 +1,145 @@
+<template>
+  <UiCard>
+    <UiCardTitle>
+      {{ t('last-n-backup-runs', 3) }}
+      <template #info>
+        <UiLink size="small" :to="`/vm/${vmId}/backups`">{{ t('backup-jobs:see-all') }}</UiLink>
+      </template>
+    </UiCardTitle>
+    <div class="backup-head">
+      <template v-if="!busy">
+        <VtsQuickInfoRow :label="t('protection-status')">
+          <template #value>
+            <UiInfo :accent="infoAccent">{{ infoText }}</UiInfo>
+          </template>
+        </VtsQuickInfoRow>
+
+        <UiButton
+          class="protection-helper"
+          accent="brand"
+          left-icon="legacy:status:info"
+          size="small"
+          variant="tertiary"
+          @click="openProtectionHelpModal()"
+        >
+          {{ t('what-does-protected-means?') }}
+        </UiButton>
```

Comment:
```suggestion
        <div class="protection-infos">
          <VtsQuickInfoRow :label="t('protection-status')">
            <template #value>
              <UiInfo :accent="infoAccent">{{ infoText }}</UiInfo>
            </template>
          </VtsQuickInfoRow>

          <UiButton
            v-if="!isEmpty || busy"
            class="protection-helper"
            accent="brand"
            left-icon="legacy:status:info"
            size="small"
            variant="tertiary"
            @click="openProtectionHelpModal()"
          >
            {{ t('what-does-protected-means?') }}
          </UiButton>
        </div>
```
I added a div class for the gap and a condition on button to display information only when there is data.